qa/tasks/parent_cache_fio.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
How is this different from the `immutable_object_cache` task above? Why do we need two?